### PR TITLE
IEC doc

### PR DESCRIPTION
**Pull request by opoudjis**

### CODE REVIEW

File: author/iec.adoc [hunk 1]
```diff
@@ -0,0 +1,25 @@
+---
+layout: iec-flavor
+title: The IEC flavor of Metanorma
+---
+:page-liquid:
+
+You can author any link:https://www.iso.org/deliverables-all.html[IEC deliverable]
```
Comment (ronaldtse) [inline author/iec.adoc]: Change to IEC deliverable link.

File: author/iec.adoc [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+with Metanorma-IEC:
+
+* International Standard
```
Comment (ronaldtse) [inline author/iec.adoc]: Change to IEC deliverable types.

**Comment by ronaldtse** [comment]
@opoudjis , the original PR contained inaccurate information. Can you replace the info with the new info we discussed? Thanks.

**Comment by opoudjis** [comment]
I do no recall what this is about, and assume it is no longer operative.

**Comment by ronaldtse** [comment]
Yup, all good, thanks.

**Pull request merged**